flashinfer/comm/trtllm_alltoall.py

```diff
@@ -309,6 +402,25 @@ def get_moe_workspaces(mapping: Mapping):
         )
         return MnnvlMoe.moe_workspace_tensor
 
+    @staticmethod
+    def get_moe_prepare_workspace(
+        mapping: Mapping, config: Optional[MnnvlConfig] = None
+    ):
+        if MnnvlMoe.moe_prepare_workspace_tensor is not None:
+            assert mapping == MnnvlMoe.moe_mapping, "only one moe mapping supported now"
+            return MnnvlMoe.moe_prepare_workspace_tensor
+        workspace_size_per_rank = get_moe_prepare_workspace_size_per_rank(
+            mapping.tp_size
+        )
+        if config:
+            MnnvlMemory.set_comm(mapping, config)
```

Comment:
`set_comm` is not defined as well

Reply:
These are added in https://github.com/flashinfer-ai/flashinfer/pull/1245